modules/javafx.graphics/src/main/java/com/sun/prism/es2/GLDrawable.java

```diff
@@ -52,4 +52,6 @@ long getNativeDrawableInfo() {
         return nativeDrawableInfo;
     }
     abstract boolean swapBuffers(GLContext glCtx);
+
+    abstract void dispose();
```

Comment:
`GLDrawable` could also get the dispose method by implementing `GraphicsResource`.

Reply:
Yes, better.